draft-ietf-quic-multipath.md

```diff
@@ -625,6 +625,29 @@ delay will most of the time relate to the path with the shortest latency.
 To collect ACK delays on all the paths, hosts should rely on time stamps
 as described in {{QUIC-Timestamp}}.
 
+### Handling ECN Feedback
```

Comment:
This text seems to be addressing the generation of ECN counters at the data receiver. How about, "Generating ACK with ECN feedback"?